settings.php

```diff
@@ -187,32 +190,40 @@
 ini_set('session.gc_divisor', 100);
 
 /**
- * Set session lifetime (in seconds), i.e. the time from the user's last visit
- * to the active session may be deleted by the session garbage collector. When
- * a session is deleted, authenticated users are logged out, and the contents
- * of the user's $_SESSION variable is discarded.
+ * Set session lifetime (in seconds).
+ *
+ * I.e. the time from the user's last visit to the active session may be deleted
+ * by the session garbage collector. When a session is deleted, authenticated
+ * users are logged out, and the contents of the user's $_SESSION variable is
+ * discarded.
  */
 ini_set('session.gc_maxlifetime', 200000);
 
 /**
- * Set session cookie lifetime (in seconds), i.e. the time from the session is
- * created to the cookie expires, i.e. when the browser is expected to discard
- * the cookie. The value 0 means "until the browser is closed".
+ * Set session cookie lifetime (in seconds).
```

Comment:
In the other cases, a noun has been used (for example *PHP settings*). If a noun is used in this case too, that should be *Session cookie lifetime* which could be followed by *(in seconds)*.